### DISCUSSION TITLE
Provide guidance on which Info Assistant resources can be cancelled or paused to control daily/monthly costs

### DISCUSSION THREAD

**Discussion by JMS206** [microsoft/PubSec-Info-Assistant#715] [Morning (EST) Office Hours (Q&A)]
Add a section to the GitHub repo that describes which resources can be metered down (or temporarily cancelled) to manage daily costs of the solution. This will be extremely helpful for subscriptions that are used for temporary self-learning or for intermittent demos for prospects/customers.

**Comment by lon-tierney** [comment]
If not ingesting new documents, you can easily stop the document processing function app. You can also adjust your app services to the free tier to reduce costs.
